collector/cloudwatch/classes/lib.php

```diff
@@ -27,17 +27,15 @@
 class lib {
     /** @var string AWS version*/
     const AWS_VERSION = '2010-08-01';
-    /** @var int Local AWS version*/
-    const LOCAL_AWS_VERSION = 2022033100;
 
     /**
      * Returns if the plugin can be used.
      *
      * @return bool
      */
     public static function is_plugin_usable() {
```

Comment:
Feels like we can remove this method as the plugin always should be usable

Reply:
I have removed this function